docs/general/oasis-network/why-oasis.md

```diff
@@ -210,7 +209,6 @@ Over 200 articles have been written since the company came out of stealth mode.
 * [The New York Times: Building a World Where Data Privacy Exists Online](https://www.nytimes.com/2019/11/19/technology/artificial-intelligence-dawn-song.html)
 * [WIRED: Oasis Labs' Dawn Song on a Safer Way to Protect Your Data](https://www.wired.com/story/dawn-song-oasis-labs-data-privacy-wired25/)
 * [MIT Technology Review: How AI could save lives without spilling medical secrets](https://www.technologyreview.com/2019/05/14/239125/how-ai-could-save-lives-without-spilling-secrets/)
-* [Yahoo! Finance: Web3 Users will Control Their Data, Says Oasis Ecosystem Head](https://www.yahoo.com/now/web3-users-control-data-says-132837055.html)
```

Comment:
I could not find one in on `archive.is` but do see Coin Rivet [here](https://coinrivet.com/web3-users-will-control-their-data-says-oasis-ecosystem-head/).

Reply:
Yeah, let's use the Coin Rivet's link.